### PR TITLE
Simplifies machinery And adds Upgrade Capabiliitiees to the cloner whilee Nurfe

### PR DESCRIPTION
**Pull request by Bod9001**
Simplifies machinery And adds Upgrade Capabiliitiees to the cloner whilee Nurfe

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/Medical/CloningPod.cs [hunk 1]
```diff
@@ -15,8 +18,19 @@ public class CloningPod : NetworkBehaviour
...
 		public CloningConsole console;
 
+		[FormerlySerializedAs("LimbCloningDamage"), SerializeField] private float internalLimbCloningDamage = 25;
+
+		[SerializeField] private float internalCloningTime = 60;
+
+
 		public float LimbCloningDamage = 25;
 
+		public float CloningTime = 60;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Medical/CloningPod.cs]: why are both ``internalCloningTime`` and ``CloningTime`` serialized? Maybe you want the later to be a property?

File: UnityProject/Assets/Scripts/Systems/Construction/MachinePartsItemTraits.cs [hunk 2]
```diff
@@ -50,5 +50,35 @@ public class MachinePartsItemTraits : SingletonScriptableObject<MachinePartsItem
 		public ItemTrait SuperCapacityPowerCell;
 		public ItemTrait HyperCapacityPowerCell;
 		public ItemTrait BluespacePowerCell;
+
+		public bool IsComponent(ItemTrait ItemTrait)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/Construction/MachinePartsItemTraits.cs:54]: similar to how ``Tier`` is part of the trait, maybe this bool should also be part of it.

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Systems/Construction/MachinePartsItemTraits.cs:54]: this is only needed for legacy support

**Comment by ThatDan123** [comment]
Codacy stuff please

**Pull request merged**